appveyor.yml

```diff
@@ -4,49 +4,85 @@ environment:
     # /E:ON and /V:ON options are not enabled in the batch script intepreter
     # See: http://stackoverflow.com/a/13751649/163740
     CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\appveyor\\run_with_env.cmd"
-    ANT_HOME: C:\ant
+    ANT_HOME: "C:\\ProgramData\\chocolatey\\lib\\ant\\apache-ant-1.10.1"
+    NUMPY_: "numpy x.x"
 
   matrix:
+    - PYTHON: "python3"
```

Comment:
Yes this is correct.  It took a while for me to get a pattern that gave cygwin the right version and kept windows the same.  It turned out the problem was that I already had mock preinstalled on my system and nosetests had a silent failure.  I kept ripping my hair out because I could not replicate the problem.

I still have a problem with the proposed patch.  I would like to autodetect the version of ant and the cygwin python version numbers.  Cygwin appends very specific version numbers like nosetests-3.6  which is great from the prospective that it we know what we have gotten, but means we have to change the version whenever cygwin updates.  I believe I can autodetect with a cygcheck | grep | sed pattern, but I was too tired to put the change in and wait 30 minutes for appveyor to run again.